tests/fixtures/mocks.py

```
@@ -277,17 +288,70 @@ def mock_flow_configure(*args, **kwargs):
+                    hasattr(self, "_discovered_devices")
+                    and device_id in self._discovered_devices
+                ):
+                    device = self._discovered_devices[device_id]
+                    if hasattr(device, "get_device_id"):
+                        device_id = device.get_device_id()
+                    if not device_name or device_name == "Unknown Device":
+                        if hasattr(device, "get_name"):
+                            try:
+                                device_name = device.get_name()
+                            except Exception:
+                                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)